src/app/hooks/quota.ts

```diff
@@ -28,10 +29,10 @@ export const useQuota = (productId: ProductType) => {
     }, [config?.ams.apiBasePath, auth]);
 
     const getQuotaTypesByProductId = () => {
-        const {
-            ams: amsConfig
-        } = config || {};
-        const { kasQuotaId, kasTrialQuotaId, srsQuotaId, srsTrialQuotaId } = amsConfig || {};
+        const kasQuotaId = Services.Kafka.Quota.QuotaId
```

Comment:
Maybe we should call it `standard` quota when more instance types will be introduced?

Reply:
The config should always be abstracted into a hook (`useConfig()`), not directly loaded like this - this will also avoid the need to `||` it. Also when loading a JSON file like that it always needs casting to a type.

Reply:
Yes. I was only correcting current code. Hook + separate package will be better